README.md

```diff
@@ -44,7 +44,7 @@ rustup target add riscv32i-unknown-none-elf
 Then, install the Nexus zkVM:
 
 ```shell
-cargo install --git https://github.com/nexus-xyz/nexus-zkvm nexus-tools --tag 'v1.0.0'
+cargo install --git https://github.com/nexus-xyz/nexus-zkvm nexus-tools --tag 'v0.2.0'
```

Comment:
Can you also add to `sdk/README.md`? I missed it in the original PR.

Reply:
added the version tag there.